verilog/CST/data_test.cc

```diff
@@ -0,0 +1,140 @@
+// Copyright 2017-2019 The Verible Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Unit tests for net-related concrete-syntax-tree functions.
+//
+// Testing strategy:
+// The point of these tests is to validate the structure that is assumed
+// about net declaration nodes and the structure that is actually
+// created by the parser, so test *should* use the parser-generated
+// syntax trees, as opposed to hand-crafted/mocked syntax trees.
+#include "verilog/CST/data.h"
+
+#include <vector>
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+#include "absl/strings/str_cat.h"
+#include "common/analysis/syntax_tree_search.h"
+#include "common/text/syntax_tree_context.h"
+#include "common/text/text_structure.h"
+#include "common/util/logging.h"
+#include "verilog/CST/verilog_nonterminals.h"
+#include "verilog/analysis/verilog_analyzer.h"
+
+#undef EXPECT_OK
+#define EXPECT_OK(value) EXPECT_TRUE((value).ok())
+
+namespace verilog {
+namespace {
+
+TEST(GetIdentifiersFromDataDeclarationTest, EmptySource) {
+  VerilogAnalyzer analyzer("", "");
+  EXPECT_OK(analyzer.Analyze());
```

Comment:
change to ASSERT_OK because there's no sense in continuing the test if this condition fails at this point.  Same in similar places below.

Reply:
Sure, changed this in the tests I'm adding to asserts.